sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -49,6 +50,7 @@ export function createFakeApplication(relations?: {
   application.currentAssessment = relations?.currentStudentAssessment;
   application.applicationNumber = faker.random.alphaNumeric(10);
   application.applicationException = relations?.applicationException;
+  application.location = relations?.location;
```

Comment:
if the location is not passed, shouldn't we create a fake location as the application status is `applicationStatus = ApplicationStatus.Submitted;` by default

Reply:
Not necessarily. As of now we can have application in submitted status and not have a location for it. We have to update the factory when we make location_id as not null.